app/controllers/api/referrral_agencies_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::ReferralAgenciesController < ApplicationController
```

Comment:
I cannot sign in right now, it just boots me out of the sign_in page if I load it